src/Framework/Assert.php

```diff
@@ -1208,6 +1208,20 @@ public static function assertNotSame($expected, $actual, $message = '')
     }
 
     /**
+     * Refutes that two variables have the same type and value.
+     * Used on objects, it refutes that two variables reference
+     * the same object. Alias for self::assertNotSame().
+     *
+     * @param mixed  $expected
+     * @param mixed  $actual
+     * @param string $message
+     */
+    public static function refuteSame($expected, $actual, $message = '')
+    {
+        return call_user_func_array('self::assertNotSame', func_get_args());
```

Comment:
Why not simply use `self::assertNotSame($expected, $actual, $message)` instead of using `call_user_func_array()`?